docs/how-to.md

```diff
@@ -19,14 +21,43 @@ SarifLog log = ... ;
 string sarifText = JsonConvert.SerializeObject(log, settings);
 File.WriteAllText(outputFilePath, sarifText);
 ```
+```C#
+# For SARIF V1 File:
+var settings = new JsonSerializerSettings()
+{
+    ContractResolver = SarifContractResolverVersionOne.Instance,
+    Formatting = Formatting.Indented
+};
+
+SarifLog log = ... ;
+
+sarifText = JsonConvert.SerializeObject(log, settings);
+File.WriteAllText(outputFilePath, sarifText);
+```
 
 ## Read a SARIF log file from disk
 
 ```C#
+# For SARIF V2 File:
 string logContents = File.ReadAllText(logFilePath);
 
 SarifLog log = JsonConvert.DeserializeObject<SarifLog>(logContents);
 ```
+```C#
+# For SARIF V1 File:
+string logContents = File.ReadAllText(logFilePath);
+
+var settings = new JsonSerializerSettings()
+{
+    ContractResolver = SarifContractResolverVersionOne.Instance
+};
+
+SarifLog log = JsonConvert.DeserializeObject<SarifLog>(logContents, settings);
```

Comment:
>SarifLog [](start = 0, length = 8)

SarifLogVersionOne``

Reply:
oh my! yes this is indeed incorrect! 

---
In reply to: [301325750](https://github.com/microsoft/sarif-sdk/pull/1566#discussion_r301325750) [](ancestors = 301325750)